### ISSUE TITLE
Add array type in Generate variables action

### ISSUE THREAD

**Issue by baptisteArno** [baptisteArno/typebot.io#1870]

**Comment by baptisteArno** [comment]
The workaround is to use the Create chat completion action and try to trick it into providing a JSON result

**Comment by bst1n** [comment]
@baptisteArno @andreanicassio the method I've found is to get the OpenAI wizard to give the result in the form of a list with the elements separated by a comma with no spaces by giving it a precise example of the type of result it should give and then to run a script immediately afterwards to format it as a list.

It's not comfortable, I'd like to be able to generate an array directly. If you think there's something better than my method, let me know here. I've struggled with this problem.

```
const sujetsGeneration = {{sujets_generation}};

if (sujetsGeneration) {
  const sujetsArray = sujetsGeneration.split(',').map(sujet => sujet.trim());
  setVariable('sujets_proposition', sujetsArray);
} else {
  setVariable('sujets_proposition', []);
}
```

**Comment by andreanicassio** [comment]
Yes that’s similar to the workaround I used. I just put in the description
of the variable to add values separated by semicolon and then do the split
with scripting.

**Comment by bst1n** [comment]
Can you share your script @andreanicassio so that I understand ? Maybe your solution is better than mine
